extra/markdown-extra/MarkdownRuntime.php

```diff
@@ -22,11 +22,41 @@ public function __construct(MarkdownInterface $converter)
 
     public function convert(string $body): string
     {
-        // remove indentation
-        if ($white = substr($body, 0, strspn($body, " \t\r\n\0\x0B"))) {
-            $body = preg_replace("{^$white}m", '', $body);
-        }
+        $body = $this->commonWhitespace($body);
+        $body = $this->removeIndentation($body);
 
         return $this->converter->convert($body);
     }
+
+    protected function commonWhitespace(string $body): string
+    {
+        return str_replace(["\t", "\0", "\x0B"], ['    ', '', ''], $body);
+    }
+
+    protected function removeIndentation(string $body): string
```

Comment:
please make those method protected. There is no reason to make them extension points of the class that would then need to be covered by BC.

Reply:
I assume you mean private?

Reply:
yeah indeed